packages/edit-site/src/components/editor/index.js

```diff
@@ -135,26 +123,28 @@ export default function Editor() {
 					} ),
 			],
 		} ),
-		[ context ]
+		[ context, setEditedPostContext ]
 	);
-	const isReady = editedPostType !== undefined && editedPostId !== undefined;
 
```

Comment:
This didn't seem to add anything over and above  `isLoaded` from  `useEditedEntityRecord`.  In my testing `editedPostType` and `editedPostId` where always defined when `isLoaded === true` , but I may have missed an edge case where this isn't the case.